### DISCUSSION TITLE
Changing node id

### DISCUSSION THREAD

**Discussion by ptorrent** [zwave-js/zwave-js#6786] [General] [answered]
Hello !

Is there a way to change a node id from the controller ? Didn't see any method for that in the documentation. Maybe it's not zwave friendly ?

THanks for your time :)

**Comment by AlCalzone** [comment, marked as answer]
Only by excluding the node and including it again a few times. After 232, the node IDs wrap back to the unused ones after 1.

There are some tricks with adding virtual nodes using PC Controller, but IMO it's not worth it.
